tests/foreman/ui/test_contentview.py

```diff
@@ -198,6 +202,8 @@ def test_positive_publish_version_changes_in_source_env(session, module_org):
         environment.
 
     :CaseLevel: Integration
+
+    :CaseImportance: high
```

Comment:
Can you capitalize this?
You've also got a number of similar lower cases in this file.

Reply:
Resolved the lower case issues on two files.